homeassistant/components/bosch_alarm/coordinator.py

```diff
@@ -0,0 +1,68 @@
+"""Coordinator for Bosch Alarm Panel."""
+
+from __future__ import annotations
+
+import logging
+
+from bosch_alarm_mode2 import Panel
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_MODEL
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class BoschAlarmCoordinator(DataUpdateCoordinator):
+    """Bosch alarm coordinator."""
+
+    def __init__(
+        self, hass: HomeAssistant, config_entry: ConfigEntry, panel: Panel
```

Comment:
```suggestion

    config_entry: BoschAlarmConfigEntry

    def __init__(
        self, hass: HomeAssistant, config_entry: BoschAlarmConfigEntry, panel: Panel
```
config entry needs to be properly typed